src/OrbitGl/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ add_library(OrbitGl STATIC)
 
 target_compile_options(OrbitGl PRIVATE ${STRICT_COMPILE_FLAGS})
 
+add_definitions(-DORBIT_GL_USE_QT)
```

Comment:
`add_definitions` is basically old-style CMake. We should use `target_compile_definitions` instead (https://cmake.org/cmake/help/v3.16/command/target_compile_definitions.html).

Like that: `target_compile_definitions(OrbitGl PRIVATE ORBIT_GL_USE_QT)`

Reply:
I've removed the definition now. Davide was also thinking of using Qt functionality in OrbitGl in a similar manner, and before we start littering the code with defines we'll just accept the fact that Qt is in there now.